src/TropicalGeometry/TropicalPolyhedron/constructors.jl

```diff
@@ -0,0 +1,151 @@
+tropical_point_configuration(P::TropicalPolyhedron{M}) where {M<:MinOrMax} = TropicalPointConfiguration{M}(pm_object(P))
+
+tropical_convex_hull(V::AbstractVector{T}) where {T<:AbstractVector{<:TropicalSemiringElem}} = tropical_convex_hull(matrix(V))
+tropical_point_configuration(V::AbstractVector{T}) where {T<:AbstractVector{<:TropicalSemiringElem}} = tropical_point_configuration(matrix(V))
+
+@doc raw"""
+    tropical_convex_hull(V::MatElem{TropicalSemiringElem{M}}) where M<:MinOrMax
+
+Constructs the tropical convex hull of the vertices `V` in the tropical projective torus
+with `M` as tropical addition.
+
+# Examples
+Defines a tropical unit ball in the 2-dimensional tropical projective torus with `max` as addition:
+```jldoctest
+julia> A = TT.(identity_matrix(QQ, 3));
+
+julia> P = tropical_convex_hull(A)
+Max tropical polyhedron in tropical projective torus of dimension 2
+```
+
+The vertices of a tropical convex hull may also be given by the rows of a tropical matrix:
+```jldoctest
+julia> TT = tropical_semiring();
+
+julia> A = TT[0 -4 -1; 0 -1 0];
+
+julia> P = tropical_convex_hull(A)
+Min tropical polyhedron in tropical projective torus of dimension 2
+```
+"""
+tropical_convex_hull(V::MatElem{TropicalSemiringElem{M}}) where {M<:MinOrMax} = _tropical_convex_hull(TropicalPolyhedron, V)
+@doc raw"""
+    tropical_point_configuration(V::MatElem{TropicalSemiringElem{M}}) where M<:MinOrMax
+
+Constructs the tropical point configuration of the vertices `V` in the tropical projective torus
+with `M` as tropical addition.
+
+# Examples
+Defines a point configuration containing a tropical unit ball in the 2-dimensional tropical projective torus with `max` as addition:
+```jldoctest
+julia> A = TT.(identity_matrix(QQ, 3));
+
+julia> P = tropical_convex_hull(A)
+Max tropical point configuration in tropical projective torus of dimension 2
+```
+The tropical point configuration may also be given by the rows of a tropical matrix:
+```jldoctest
+julia> TT = tropical_semiring();
+
+julia> A = TT[0 -4 -1; 0 -1 0];
+
+julia> P = tropical_point_configuration(A)
+Min tropical point configuration in tropical projective torus of dimension 2
+```
+"""
+tropical_point_configuration(V::MatElem{TropicalSemiringElem{M}}) where {M<:MinOrMax} = _tropical_convex_hull(TropicalPointConfiguration, V)
+
+@doc raw"""
+    tropical_convex_hull(convention::MinOrMax, V::QQMatrix)
+
+Constructs the tropical convex hull of the vertices `V` in the tropical projective torus
+with `convention` as tropical addition.
+
+# Examples
+Defines a point configuration containing a tropical unit ball in the 2-dimensional tropical projective torus with `max` as addition:
+```jldoctest
+julia> A = identity_matrix(QQ, 3);
+
+julia> P = tropical_convex_hull(max, A)
+Max tropical polyhedron in tropical projective torus of dimension 2
+```
+"""
+tropical_convex_hull(convention::MinOrMax, V::QQMatrix) = _tropical_convex_hull(TropicalPolyhedron, convention, V)
```

Comment:
```suggestion
tropical_convex_hull(V::QQMatrix, convention::MinOrMax=min) = _tropical_convex_hull(TropicalPolyhedron, convention, V)
```

Can you please put `convention` last, make it optional with default min?  

I think there might an unfortunate clash of conventions between the polyhedral part and the tropical part of OSCAR: input which specifies the return type is listed first in the polyhedral part while conventions and tropical semiring maps are listed last in the tropical part (because I'm used to C where optional input has to be last).

Reply:
For the sake of consistency, can you please change all user-facing functions accordingly?  (I'm fine with the non-user facing functions staying the way they are)